StyleCop.Analyzers/StyleCop.Analyzers/Lightup/LightupHelpers.cs

```diff
@@ -280,5 +292,19 @@ internal static bool CanWrapNode(SyntaxNode node, Type underlyingType)
                     valueParameter);
             return expression.Compile();
         }
+
+        private static bool ValidatePropertyType(Type returnType, Type actualType)
+        {
+            if (returnType.GetTypeInfo().ImplementedInterfaces.Any(t => t.GetGenericTypeDefinition() == typeof(ISyntaxWrapper<>)))
+            {
+                // The requested property is a wrapper type, so validate against the wrapped type.
+                var wrappedType = WrapperHelper.GetWrappedType(returnType);
+                return wrappedType == actualType;
+            }
+            else
+            {
+                return returnType == actualType;
```

Comment:
:memo: It doesn't appear this exception case is covered by any tests (looking at codecov)